### ISSUE TITLE
File Links

### ISSUE THREAD

**Issue by jposeyjr** [kidscancode/godot_recipes#124]
The file links don't seem to be working anymore and even looking through the repo I only see 3 file zips.

**Comment by R366Y** [comment]
I agree with @jposeyjr , it would be very nice to have source code of the examples as links or here in github.
Thank you for the amazing tutorials btw.
